examples/eSealing/src/eSealing.tsx

```diff
@@ -0,0 +1,413 @@
+                {!activeConnectorError && !isWaitingForTransaction && activeConnectorType && !activeConnector && (
+                    <p>
+                        <i>Loading connector...</i>
+                    </p>
+                )}
+                {!activeConnection && !isWaitingForTransaction && activeConnectorType && activeConnector && (
+                    <p>
+                        <button style={ButtonStyle} type="button" onClick={connect}>
+                            {isActiveConnectorWaitingForUser && 'Connecting...'}
+                            {!isActiveConnectorWaitingForUser &&
+                                activeConnectorType === 'BrowserWallet' &&
+                                'Connect Browser Wallet'}
+                            {!isActiveConnectorWaitingForUser &&
+                                activeConnectorType === 'WalletConnect' &&
+                                'Connect Mobile Wallet'}
+                        </button>
+                    </p>
+                )}
+                {activeConnectedAccount && (
+                    <>
+                        <div className="text">Connected to</div>
+                        <button
+                            className="link"
+                            type="button"
+                            onClick={() => {
+                                window.open(
+                                    `https://testnet.ccdscan.io/?dcount=1&dentity=account&daddress=${activeConnectedAccount}`,
+                                    '_blank',
+                                    'noopener,noreferrer'
+                                );
+                            }}
+                        >
+                            {activeConnectedAccount}
+                        </button>
+                        <br />
+                        <br />
+                        <div className="containerSpaceBetween">
+                            <button
+                                style={
+                                    activeConnectedAccount === undefined || isRegisterFilePage === false
+                                        ? ButtonStyleDisabled
+                                        : ButtonStyle
+                                }
+                                type="button"
+                                disabled={activeConnectedAccount === undefined}
+                                onClick={() => {
+                                    setIsRegisterFilePage(true);
+                                    setFileHash('');
+                                    setFileHashHex('');
+                                    setWitness('');
+                                    setTimestamp('');
+                                    setHash('');
+                                }}
+                            >
+                                Register File Hash
```

Comment:
It took me a moment to understand that this button doesn't actually register the hash, only selects that function. And it's a little confusing that there is another button with the same text.

Maybe make it a `<label>` instead?

Reply:
Can you elaborate on how I should use the `<label>` tag here?

I changed the description of the buttons to `Registration Tab` and `Display Tab` and also changed the colors to improve the understanding of this switch.

Reply:
I was just thinking of rendering it as text instead of a button along the lines of how `<label>` is used for form fields. I don't know whether `<label>` can switch the toggle natively the same way here, but it might at least give you the "correct" styling for free. You can always use the same click handler like you do now, regardless of what HTML element you use.

But I don't think you need to change it, the current solution is fine with me.